.github/workflows/version_bump.yml

```diff
@@ -0,0 +1,31 @@
+name: Version Bump
+
+on:
+  create: # It will trigger whenever a new tag or branch is created
+
+jobs:
+  version-bump:
+    if: startsWith(github.ref, 'refs/tags/') # true if new tag is created
+    runs-on: ubuntu-22.04
+    timeout-minutes: 3
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+
+      - name: Install dependencies
+        run: pnpm install
```

Comment:
I think we can get away without installing all of the packages, just need to install `pnpm` using `- uses: pnpm/action-setup@v2`